src/test/scala/uk/gov/nationalarchives/consignmentexport/MainSpec.scala

```diff
@@ -146,7 +147,7 @@ class MainSpec extends ExternalServiceSpec {
     secondEmptyDirectory.list().length should be (0)
   }
 
-  "the export job" should "update the export data in the api for a 'standard' consignment type" in {
+  "the export job" should "update the export data and update the consignment status as Completed in the api for a 'standard' consignment type" in {
```

Comment:
Maybe put `Completed` in quotes to make the test description a little clearer: `... status as 'Completed' in the api ...`